f5_sphinx_theme/head.html

```diff
@@ -17,13 +17,15 @@
   <!-- Twitter Bootstrap viewport setting -->
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <!-- F5 metadata -->
-  <meta name="kb_doc_type" content="Manual"/>
-  <meta name="kb_doc_type" content="User Guide"/>
   <meta name="title" content="{{ title|striptags|e }}"/>
-  <meta name="product" content="{{ project|striptags|e }}"/>
+  <meta name="product" content="{{ project|striptags|e }}"/> -
   <meta name="version" content="{{ release|striptags|e }}"/>
-  <meta name="updated_date"  content="{{ last_updated|e }}"/>
-  <meta name="archived" content="Archived documents excluded"/>
+  <meta name="updated_date"  content="{{ last_updated|e }}"/> -
+  <meta name="archived" content="Archived documents excluded"/> -
+  <meta name="doc_type" content="Manual"/>
+  <meta name="lifecycle" content="release"/>
+  <meta name="F5 Cloud Services" content="1.0"/>
```

Comment:
Oh, this is not to be hard-coded--
  <meta name="F5 Cloud Services" content="1.0"/>
It's supposed to be variables. See my email.

Reply:
Gotcha - working on fixing it now...

Reply:
This is fixed now; please review and let me know if there is anything else needs to be adjusted.